include/ck/tensor_operation/gpu/device/impl/device_gemm_xdl_cshuffle_v3.hpp

```diff
@@ -21,6 +21,104 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 
+/// @brief \"Universal\" GEMM operation with SplitK support.
+///
+/// @par Overview
+///         This GEMM operation is carrying out following mathematical equation:
+///         C{M,N} = C_op(A_op(A{M,K}) * B_op(B{K,N}))
+///         Where A, B are input tensors and C is the output tensor. The A/B/C_op are
+///         elementwise operations that could be applied on each tensor respectively.
```

Comment:
```suggestion
///         elementwise operations applied to the A, B, and C tensors, respectively.
```

I was a bit confused here by "could". I assume by that it's meant that the ops are optional and not required.  But I think that goes without saying. Or should, when discussion a GEMM.

Reply:
Actually I've deliberately used it in order to emphasize that those operations are optional :)